frontend/src/pages/projects/screens/detail/notebooks/NotebookImageDisplayName.tsx

```diff
@@ -190,13 +191,7 @@ export const NotebookImageDisplayName = ({
                     variant="outline"
                     color="blue"
                     data-testid="project-scoped-label"
-                    icon={
-                      <img
-                        style={{ height: '20px' }}
-                        src={typedObjectImage(ProjectObjectType.project)}
-                        alt=""
-                      />
-                    }
+                    icon={<TypedObjectIcon alt="" resourceType={ProjectObjectType.project} />}
```

Comment:
Looking at the doc mentioned in the issue, when the notebook image name is short -  this is how it looks like.
<img width="1421" alt="Screenshot 2025-05-07 at 4 21 41 PM" src="https://github.com/user-attachments/assets/3cd6b6c2-be7a-4970-98c0-15bd80964fbf" />